src/components/Match/matchColumns.jsx

```diff
@@ -51,6 +52,13 @@ export const heroTdColumn = {
   sortFn: true,
 };
 
+export const heroTdChatColumn = {
```

Comment:
So I actually wasn't sure of a way to do that. Was not aware that you could use rest in that way. TIL